content/build/hydrobuild.md

```diff
@@ -263,39 +260,87 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: .
-          push: true
           tags: user/app:latest
+          # if this runs in a pull request, export results to build cache
+          outputs: ${{ github.event_name == 'pull_request' && 'type=cacheonly' || '' }}
+          # if this doesn't run in a pull request, push to a registry
+          push: ${{ github.event_name != 'pull_request' }}
 ```
 
 This invokes the build from a GitHub Actions workflow, runs the build on
 Hydrobuild, and pushes the image to a Docker Hub registry.
 
-> **Note**
->
-> The previous example uses a `push: true` configuration for the _Build and
-> push_ GitHub Action. This ensures that the build result is pushed to a
-> registry directly, rather than being loaded back to the image store of the
-> GitHub Actions runner. When using Hydrobuild in CI, this is the recommended
-> workflow, because it speeds up your builds and avoids unnecessary file
-> transfers.
->
-> If you're not using `push: true`, and if you build an image with a `tag`,
-> Hydrobuild automatically loads the build results back to the client. If you
-> only want to build the artifact without loading the results (as a validation
-> step in pull requests, for example), you can add `outputs: type=cacheonly` to
-> the action configuration:
->
-> ```yaml
-> - name: Build and push
->   uses: docker/build-push-action@v4
->   with:
->     context: .
->     tags: user/app:latest
->     # if this runs in a pull request, export results to build cache
->     outputs: ${{ github.event_name == 'pull_request' && 'type=cacheonly' || '' }}
->     # if this doesn't run in a pull request, push to a registry
->     push: ${{ github.event_name != 'pull_request' }}
-> ```
+{{< /tab >}}
+{{< tab name="Circle CI" >}}
+
+```yaml
+version: 2.1
+
+jobs:
+  setup:
+    docker:
+      - image: cimg/base:2023.08
+    steps:
+      - run: mkdir -vp ~/.docker/cli-plugins/
+      - run: curl --silent -L --output ~/.docker/cli-plugins/docker-buildx https://github.com/docker/buildx-desktop/releases/download/v0.11.2-desktop.4/buildx-v0.11.2-desktop.4.linux-amd64
+      - run: chmod a+x ~/.docker/cli-plugins/docker-buildx
+
+      - run: echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
+      - run: docker buildx create --use --driver cloud --name ripc-builder ripc/blazers
```

Comment:
Remove `--name ripc-builder` as `--use` will make it default
Change `ripc/blazers` to `<org>/<group>` to match the generic placeholders in GHA example